app/models/bike.rb

```diff
@@ -466,6 +463,30 @@ def normalize_attributes
     true
   end
 
+  def normalize_serial_number
+    if made_without_serial?
+      self.serial_number = "made_without_serial"
+      self.serial_normalized = nil
+      return true
+    end
+
+    self.serial_number = SerialNormalizer.unknown_and_absent_corrected(serial_number)
+
+    case serial_number
+    when "made_without_serial"
```

Comment:
nit: might be cleaner to add `made_without_serial?` to this `when` and then remove the above if statement. But doesn't really matter.

Reply:
That would evaluate differently, and the semantics of the case statement make it harder to reason about:

```rb
def case_statement(serial_number, made_without_serial:)
  case serial_number
  when "made_without_serial", made_without_serial
    :made_without_serial
  else
    :made_with_serial
  end
end

case_statement("made_without_serial", made_without_serial: false)
# :made_without_serial

case_statement("serial_number", made_without_serial: true)
# :made_with_serial
```